.github/actions/simple-md-toc/action.yml

```diff
@@ -0,0 +1,57 @@
+#Github Composite actions to generate Markdown Table of Contents
+name: Simple Markdown ToC Generator
+description: Generate a Table of Contents from a Markdown file
+inputs:
+  md_toc_version:
+    description: Markdown Table of Contents Package version
+    required: false
+    default: 1.2.0
+  md_src:
+    description: Markdown file to generate Table of Contents
+    required: true
```

Comment:
it supports only one?

Reply:
yes it seems so. I can add support for passing a space separated list of files? or maybe a json string where one can put different options for different files?

Reply:
for now may be fine, maybe in the future we need to pass a glob and enumerate them (pre-commit would had handled it automatically xD)

Reply:
Have a look if that's suits you now.